doc/user_guide/dynamo0p3.rst

```diff
@@ -313,6 +313,17 @@ For example::
   ! ...
   call invoke(kernel(field1, field2, extent, x_direction))
 
+If the stencil is of type ``CROSS2D`` then the arrays passed to the kernel are
+of different dimensions to those of other stencils. The ``CROSS2D`` stencil is
+designed for use when knowing where the stencil cells are in relation to the
```

Comment:
"...for use when it is necessary for a kernel to know where the stencil cells are, relative to the current cell."

Reply:
Changed.